app/configurator/components/field.tsx

```diff
@@ -53,6 +55,15 @@ import { IconName } from "@/icons";
 import { useLocale } from "@/locales/use-locale";
 import { getPalette } from "@/palettes";
 
+const useStyles = makeStyles<Theme>((theme) => ({
+  loadingIndicator: {
+    color: theme.palette.grey[700],
+    display: "inline-block",
+    marginTop: 3,
+    marginLeft: 8,
```

Comment:
Nit: Shouldn't those margins be defined as function of theme.spacing ?

Reply:
Yes, thanks for noticing this 👍